.circleci/config.yml

```diff
@@ -1,10 +1,20 @@
 version: 2.1
 orbs:
   docker: circleci/docker@0.5.19
+  helm-tools: pennlabs/helm-tools@0.1.1
 workflows:
   version: 2
   build-and-publish:
     jobs:
+      - helm-tools/helm-publish:
```

Comment:
I think we should move this job after `docker/publish` and require that job to finish first.

Reply:
oh nice